packages/react-instantsearch-dom/src/core/createInstantSearchServer.js

```diff
@@ -150,28 +161,38 @@ export const findResultsState = function(App, props) {
 
   const { indexName, searchClient } = props;
 
-  const searchParameters = [];
+  const widgets = [];
 
   renderToString(
-    <App
-      {...props}
-      onSearchParameters={createSearchParametersCollector(searchParameters)}
-    />
+    <App {...props} widgetsCollector={createWidgetsCollector(widgets)} />
```

Comment:
Don't forget to update [this doc](https://www.algolia.com/doc/guides/building-search-ui/going-further/server-side-rendering/react/) to adapt to `widgetsCollector`.

Reply:
https://github.com/algolia/doc/pull/4992